### ISSUE TITLE
Collections/whitelist should override player property flag

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#132]
Nexus report:
> I did run into one thing that didn't let looted, although it should be in collections, the Golden human hand (SUT) at solitude apothecary.
> Edit: burnt (helgen cave) from badgremlins also didn't get looted. Both burnt and golden human hand did get looted once I put my player owned settings to take all.

All other things being equal, an item that is on whitelist or in a collection should be picked up even it is player property.

**Comment by SteveTownsend** [comment]
Two changes under test here:
1. reset state properly after various events - timed update, game reload and MCM usage. This should fix "PlayerOwned' stickiness.
2. modify item-level Legality check to Collect the item if it's owned but not a Crime. This makes Collections work the way I want.
Under test for v1.10.0.0

**Comment by SteveTownsend** [comment]
Test evidence for cases above, with Burnt. In each case, exiting the MCM while standing by the unlooted item causes the item to be looted.
1. stand by the item and turn on Player-Owned
```
J:\GitHub\SmartHarvestSE\src\Looting\ScanGovernor.cpp(250): [VERBOSE] {26596} (2020-07-19 03:35:25 PM) Process REFR 0x0a0008a5 with base object Burnt/0x0a00081d
J:\GitHub\SmartHarvestSE\src\Utilities\debugs.cpp(155): [VERBOSE] {26596} (2020-07-19 03:35:25 PM) 0x0a00081d 0x20(32) [Burnt] - typename clutter
J:\GitHub\SmartHarvestSE\src\Utilities\debugs.cpp(147): [MESSAGE] {26596} (2020-07-19 03:35:25 PM) Worth(200.00)  Weight(2.00)  V/W(100.00)
J:\GitHub\SmartHarvestSE\src\Looting\TryLootREFR.cpp(767): [VERBOSE] {26596} (2020-07-19 03:35:25 PM) Player-owned true, looting belongings disallowed: Burnt/0x0a00081d
```
```
J:\GitHub\SmartHarvestSE\src\Looting\ScanGovernor.cpp(250): [VERBOSE] {26596} (2020-07-19 03:35:25 PM) Process REFR 0x0a0008a5 with base object Burnt/0x0a00081d
J:\GitHub\SmartHarvestSE\src\Utilities\debugs.cpp(155): [VERBOSE] {26596} (2020-07-19 03:35:25 PM) 0x0a00081d 0x20(32) [Burnt] - typename clutter
J:\GitHub\SmartHarvestSE\src\FormHelpers\IHasValueWeight.cpp(50): [VERBOSE] {26596} (2020-07-19 03:35:25 PM) Checking value: 200
J:\GitHub\SmartHarvestSE\src\FormHelpers\IHasValueWeight.cpp(97): [VERBOSE] {26596} (2020-07-19 03:35:25 PM) Burnt/0x0a00081d item VW 100.00 vs threshold VW 10.00
J:\GitHub\SmartHarvestSE\src\Looting\TryLootREFR.cpp(289): [VERBOSE] {26596} (2020-07-19 03:35:25 PM) Burnt/0x0a00081d value:200
J:\GitHub\SmartHarvestSE\src\Looting\TryLootREFR.cpp(318): [VERBOSE] {26596} (2020-07-19 03:35:25 PM) SmartHarvest Burnt/0x0a00081d for REFR 0x0a0008a5
```
2. stand by the item and turn on Collections
```
J:\GitHub\SmartHarvestSE\src\Looting\ScanGovernor.cpp(250): [VERBOSE] {26596} (2020-07-19 03:38:11 PM) Process REFR 0x0a0008a5 with base object Burnt/0x0a00081d
J:\GitHub\SmartHarvestSE\src\Utilities\debugs.cpp(155): [VERBOSE] {26596} (2020-07-19 03:38:11 PM) 0x0a00081d 0x20(32) [Burnt] - typename clutter
J:\GitHub\SmartHarvestSE\src\Looting\TryLootREFR.cpp(154): [VERBOSE] {26596} (2020-07-19 03:38:11 PM) Collectible Item 0x0a00081d
J:\GitHub\SmartHarvestSE\src\Looting\TryLootREFR.cpp(767): [VERBOSE] {26596} (2020-07-19 03:38:11 PM) Player-owned true, looting belongings disallowed: Burnt/0x0a00081d
J:\GitHub\SmartHarvestSE\src\Looting\TryLootREFR.cpp(736): [VERBOSE] {26596} (2020-07-19 03:38:11 PM) Collectible REFR 0x0a0008a5 overrides Legality PlayerOwned for Burnt/0x0a00081d
J:\GitHub\SmartHarvestSE\src\Looting\TryLootREFR.cpp(223): [VERBOSE] {26596} (2020-07-19 03:38:11 PM) check REFR to collectible 0x0a00081d
J:\GitHub\SmartHarvestSE\src\Looting\TryLootREFR.cpp(318): [VERBOSE] {26596} (2020-07-19 03:38:11 PM) SmartHarvest Burnt/0x0a00081d for REFR 0x0a0008a5
```

**Comment by SteveTownsend** [comment]
fixed in v1.10.0.0
